receiver/podmanreceiver/receiver.go

```diff
@@ -12,33 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build !windows
+
 package podmanreceiver
 
 import (
 	"context"
 	"fmt"
 	"net/url"
+	"time"
 
+	"github.com/containers/podman/v2/libpod/define"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/obsreport"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver/interval"
```

Comment:
Can this dependency be avoided?

Reply:
Yes, by copying what it does :) I think we should move it out of redisreceiver to the "global" internal package.